api/app/utils.py

```diff
@@ -1,7 +1,11 @@
+import json
 import pathlib
 
 import shortuuid
 
+UNKNOWN = "unknown"
+VERSIONS_INFO_FILE_LOCATION = ".versions.json"
```

Comment:
Not really sure where this file path has come from? It should be `.release-please-manifest.json` or, reiterating my previous comment, `version.txt` which would some of the code here?

Reply:
Ah, just remembered it's part of the docker build I think? I still think it would be better to use the `version.txt` file though.

Reply:
Yep the modified Dockerfiles do this. I think its better to stay as json as it's easier to parse across both JS and Python.